### DISCUSSION TITLE
waline文档是不是应该增加v2升级v3，以及不要把v2相关文档全部删除了 || Waline documents should be added to v2 to upgrade to v3, and should not delete all v2 related documents

### DISCUSSION THREAD

**Discussion by appotry** [walinejs/waline#2372] [General]
原来使用waline v2版本，采用了IntersectionObserver 技术来实现延迟加载

v3版本的client 库，还没想好怎么和原来的代码兼容

对文档的建议

- v2 版本 ，v3版本的文档分开
- v2 与v3的差异，升级注意事项，外在css , js 选择器使用升级注意事项等
- v2如何升级到v3
- v3 设计思路，与v2的优势
- 等等

---
Originally using the waline v2 version, IntersectionObserver technology was used to implement lazy loading.

The v3 version of the client library has not yet been figured out how to be compatible with the original code.

Suggestions for documentation

- v2 version, v3 version documents are separated
- Differences between v2 and v3, upgrade precautions, external css, js selector usage upgrade precautions, etc.
- How to upgrade v2 to v3
- v3 design ideas and advantages of v2
- Wait

**Comment by Mister-Hope** [comment]
https://waline.js.org/migration/v3.html

总体来说，由于浏览器对 ESM 支持比例提高以及所有的 Node 版本都支持 ESM，所以不再提供 CJS 和 UMD 版本的文件。其他变更详见迁移指南。

---
https://waline.js.org/migration/v3.html

In general, due to the increase in browser support for ESM and the fact that all Node versions support ESM, the CJS and UMD versions of the files are no longer provided. Other changes are detailed in the migration guide.
